### PR TITLE
fix:[UI] Align Card title with content

### PR DESCRIPTION
**Pull request by somchi**
fix: solving issue with card title with content not aligned in localhost:5080/provisioners

Closes https://github.com/taskcluster/taskcluster/issues/1750.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,8 +1,11 @@
 node_modules
 .eslintcache
+.vscode
 user-build-config.yml
 *.log
 .idea
 user-config.*
 dev-config.*
 /artifacts/
+/node_modules
```
Comment (vishakhanihore) [inline .gitignore]: It is not required, The following are already ignored.

Comment (helfi92) [inline .gitignore]: Agreed, let's remove `/node_modules`. This is covered by the first line `node_modules`.

File: ui/src/components/ProvisionerDetailsCard/index.jsx [hunk 2]
```diff
@@ -30,8 +30,8 @@ import { ACTION_CONTEXT } from '../../utils/constants';
     fontSize: '0.7rem',
   },
   headline: {
-    paddingLeft: theme.spacing.triple,
-    paddingRight: theme.spacing.triple,
+    paddingLeft: theme.spacing.double,
+    paddingRight: theme.spacing.double,
```
Comment (helfi92) [inline ui/src/components/ProvisionerDetailsCard/index.jsx]: Nice 💯

File: .gitignore (hunk 1 above)
Comment (somchi) [reply .gitignore]: alright i will fix that

**Comment by somchi** [comment]
is it fine this way?

File: .gitignore [hunk 3]
```diff
@@ -1,8 +1,10 @@
 node_modules
 .eslintcache
+.vscode
 user-build-config.yml
 *.log
 .idea
 user-config.*
 dev-config.*
 /artifacts/
+/ui/node_modules
```
Comment (helfi92) [inline .gitignore]: `/ui/node_modules`  still needs to go away.

Comment (somchi) [reply .gitignore]: will do that right away

**Comment by somchi** [comment]
thanks, that's fine

**Comment by somchi** [comment]
can i add it to my list of contributions now, and since i there are few days left to the end of the contribution stage, is there any other work i can still try my hands on

**Comment by helfi92** [comment]
Definitely. Please do!

**Comment by helfi92** [comment]
I've assigned you to https://github.com/taskcluster/taskcluster/issues/1774.

**Comment by helfi92** [comment]
@all-contributors please add @somchi for code.

**Comment by helfi92** [comment]
Done in https://github.com/taskcluster/taskcluster/pull/1846#pullrequestreview-318580648.

**Pull request closed without merging**